src/testsuite/vec_f64_dummy.c

```diff
@@ -561,6 +561,7 @@ __test_cmpledp (vf64_t a, vf64_t b)
 {
   return vec_cmple (a, b);
 }
+#endif
```

Comment:
That's quite a long move. Maybe explain why in the commit message? It seems unlikely that you added 150 lines of code because of `Wall` or typos.

Reply:
The code change is good but the change-log is wrong. For vec_f64_dummy.c I need to guard the vector double compare test but keep the gather/scatter matrix transposed tests enabled for pveclib_perf measurements.